### PR TITLE
[CBRD-22990] Fixed sysop in select incr

### PR DESCRIPTION
**Pull request by alexstanro**
http://jira.cubrid.org/browse/CBRD-22990

The issue is caused by the fact that transaction was not aborted.

### CODE REVIEW

File: src/query/query_executor.c [hunk 1]
```diff
@@ -12651,22 +12651,35 @@ qexec_execute_selupd_list (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XASL_STATE
...
+  if (lock_is_instant_lock_mode (tran_index))
+    {
+      /* Subtransaction case. */
+      assert (subtransaction_started);
       log_sysop_commit (thread_p);
     }
+  else
+    {
+      /* Transaction case. */
+      log_sysop_attach_to_outer (thread_p);
```
Comment (ribram) [inline src/query/query_executor.c]: might help to add:
```c
    assert (!subtransaction_started);
```

Comment (alexstanro) [reply src/query/query_executor.c]: It is not correct, but I also did this mistake initially. Sub transaction may be started (initially instant lock mode is true). Then, during qexec_execute_increment instant lock mode is reset to false (due to lock_object calls). In this moment we have subtransaction started with instant lock mode false.

Comment (ribram) [reply src/query/query_executor.c]: hm, instant lock stop during qexec_execute_increment causes inconsistencies. are you sure it is possible?

if it is true, what happens with the sub-mvcc?

Comment (alexstanro) [reply src/query/query_executor.c]: Sure, locks in btree for instance. Sub-mvcc is completed in this case.

**Pull request merged**